bucket/electerm.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "1.34.2",
+    "description": "Terminal/ssh/serial port/sftp client",
+    "homepage": "https://electerm.github.io/electerm/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/electerm/electerm/releases/download/v1.3.42/electerm-1.3.42-win-x64.tar.gz",
+            "hash": "a5ca67755cd078c368fa19f1422cb77683e40e33516cb147fbd227327845f425",
+            "extract_dir": "electerm-1.3.42-win-x64"
+        }
+    },
+    "shortcuts": [
```

https://github.com/electerm/electerm/blob/c3de2ed9a6636db524b2ec420c62cdb0d74e4a3d/src/app/app.js#L106-L116

```suggestion
    "bin": "electerm.exe",
    "shortcuts": [
```